internal/impl/amqp09/input.go

```diff
@@ -111,7 +112,7 @@ then the declaration passively verifies that they match the target fields.`,
 	}
 }
 
-//------------------------------------------------------------------------------
+// ------------------------------------------------------------------------------
```

Comment:
My IDE (IntelliJ, has formal Go support) basically forced these comment formatting changes (I think it runs `gofmt` any time I save).

Is that okay?

Reply:
I doubt it's `gofmt` as I've never seen it happen before, I'd be happier just deleting the lines if you can't prevent your IDE from changing it.